planet/data/orders_product_bundle_2023-02-24.json

```diff
@@ -574,56 +491,15 @@
         ]
       }
     },
-    "basic_uncalibrated_dn_nitf": {
-      "name": "Basic Uncalibrated DN - NITF",
-      "description": "Uncalibrated digital numbers - includes RPC text file for georeferencing",
-      "bands": "all",
-      "rectification": "non-orthorectified",
-      "radiometry": "digital numbers",
-      "fileType": "NITF",
-      "auxiliaryFiles": "udm",
-      "assets": {
-        "PSScene4Band": [
-          "basic_analytic_dn_nitf",
-          "basic_analytic_dn_xml_nitf",
-          "basic_analytic_dn_rpc_nitf",
-          "basic_udm"
-        ]
-      }
-    },
-    "basic_uncalibrated_dn_nitf_udm2": {
-      "name": "Basic Uncalibrated DN - NITF",
-      "description": "Uncalibrated digital numbers - includes RPC text file for georeferencing",
-      "bands": "all",
-      "rectification": "non-orthorectified",
-      "radiometry": "digital numbers",
-      "fileType": "NITF",
-      "auxiliaryFiles": "udm, udm2",
-      "assets": {
-        "PSScene4Band": [
-          "basic_analytic_dn_nitf",
-          "basic_analytic_dn_xml_nitf",
-          "basic_analytic_dn_rpc_nitf",
-          "basic_udm",
-          "basic_udm2"
-        ]
-      }
-    },
     "basic_analytic_nitf": {
       "name": "Basic Analytic Radiance (TOAR) – NITF",
-      "description": " Calibrated to top of atmosphere radiance - includes RPC text file for georeferencing",
+      "description": "Calibrated to top of atmosphere radiance - includes RPC text file for georeferencing",
```

Comment:
Good catch on the whitespace @JuLeeAtPlanet  ! I don't know what to think about the single dash there :smile: We could have an em-dash in JSON... but maybe I'm just being obsessive.